src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -97,6 +49,60 @@ public class ConfigurationKeys
     /// </summary>
     public const string FlushOnUnhandledException = "OTEL_DOTNET_AUTO_FLUSH_ON_UNHANDLEDEXCEPTION";
 
+    /// <summary>
+    /// Configuration keys for trace exporter
+    /// </summary>
+    public static class Tracer
+    {
+        /// <summary>
+        /// Configuration key for enabling or disabling the Tracer.
+        /// Default is value is true (enabled).
+        /// </summary>
+        /// <seealso cref="Settings.TraceEnabled"/>
+        public const string TraceEnabled = "OTEL_DOTNET_AUTO_TRACE_ENABLED";
+
+        /// <summary>
+        /// Configuration key for whether the tracer should be initialized by the profiler or not.
+        /// </summary>
+        public const string LoadTracerAtStartup = "OTEL_DOTNET_AUTO_LOAD_TRACER_AT_STARTUP";
+
+        /// <summary>
+        /// Configuration key for the traces exporter to be used.
+        /// Default is <c>"otlp"</c>.
+        /// </summary>
+        public const string TracesExporter = "OTEL_TRACES_EXPORTER";
+
+        /// <summary>
+        /// Configuration key for whether the trace console exporter is enabled.
+        /// </summary>
+        public const string ConsoleExporterEnabled = "OTEL_DOTNET_AUTO_TRACE_CONSOLE_EXPORTER_ENABLED";
```

Comment:
I think we should rename `OTEL_DOTNET_AUTO_TRACE_CONSOLE_EXPORTER_ENABLED` to `OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`.

The same comment applies to other places - we should rename rename `_TRACE_` to `_TRACES_`

Reference: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md

Reply:
Should we use `OTEL_DOTNET_AUTO_LOAD_TRACER_AT_STARTUP` or `OTEL_DOTNET_AUTO_LOAD_TRACES_AT_STARTUP`?

Reply:
Fixed dc8ce26a0e501cebef0b66399b0826ec094755ee